### DISCUSSION TITLE
Keep getting rejected when everything is provided

### DISCUSSION THREAD

**Discussion by jaredgizersky** [community/community#55781] [GitHub Education]
Hi everyone,
As the title says, I keep getting rejected with the following message: Your application cannot be reviewed until you fix the following: You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio. I filled out my bio with my school and graduation year (removed now for privacy), but it gave me the same message as before. What am I meant to do here? It auto-rejected me despite filling out a bio and my full name.

**Comment by samus-aran** [comment]
Hi @jaredgizersky 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
